CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ Copy and pasting the git commit messages is __NOT__ enough.
 - `history_vehicles_replacement_for_imitation_learning.py` now uses new Imitation action space.  See Issue #844.
 ### Fixed
 - Allow for non-dynamic action spaces to have action controllers.  See PR #854.
-
+- Fixed a bug in `sensors.py` which triggered `wrong_way` event when the vehicle goes into an intersection. See Issue #846.
```

Comment:
we'll probably want to keep the newline between version blocks.

Reply:
ok sure, will add these changes.